### PR TITLE
Add Estonian language

### PR DESCRIPTION
**Pull request by kukrik**
Add Estonian language

### CODE REVIEW

File: reportwidgets/browsers/partials/_widget.htm [hunk 1]
```diff
@@ -9,18 +9,18 @@ <h3><?= e($this->property('title')) ?></h3>
             data-full-width="1"
         >
             <ul>
-            <?php if (count($rows)): ?>
```
Comment (LukeTowers) [inline reportwidgets/browsers/partials/_widget.htm]: Please keep the original indentation

Comment (kukrik) [reply reportwidgets/browsers/partials/_widget.htm]: I'll take note of your recommendation. I do not do stupid things continue to...
By the way, I'm a new boy here for one month.

**Comment by LukeTowers** [comment]
Look at https://github.com/rainlab/googleanalytics-plugin/pull/80/files to see the difference in your changes vs the original, the indentation in your changes is still different from the original.

**Comment by bennothommo** [comment]
@kukrik I have reverted your browser widget changes and will just merge the Estonian language file for now. If you would like to implement your fix to the widget, please open a new PR, taking care to ensure that the indentation of the code matches the original file.

Thanks for your contribution.

**Pull request merged**